public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx

```diff
@@ -120,61 +96,38 @@ class CloudWatchLogsQueryField extends React.PureComponent<CloudWatchLogsQueryFi
     );
   };
 
-  render() {
-    const { onRunQuery, onChange, ExtraFieldElement, data, query, datasource, theme } = this.props;
-    const { expression } = query;
-    const { hint } = this.state;
-
-    const showError = data && data.error && data.error.refId === query.refId;
-    const cleanText = datasource.languageProvider ? datasource.languageProvider.cleanText : undefined;
-
-    return (
-      <>
-        <QueryHeader
-          query={query}
-          onRunQuery={onRunQuery}
-          datasource={datasource}
-          onChange={onChange}
-          sqlCodeEditorIsDirty={false}
-        />
-        <LogGroupSelection datasource={datasource} query={query} onChange={onChange} onRunQuery={onRunQuery} />
-        <div className="gf-form-inline gf-form-inline--nowrap flex-grow-1">
-          <div className="gf-form gf-form--grow flex-shrink-1">
-            <QueryField
-              additionalPlugins={this.plugins}
-              query={expression ?? ''}
-              onChange={this.onChangeQuery}
-              onRunQuery={this.props.onRunQuery}
-              onTypeahead={this.onTypeahead}
-              cleanText={cleanText}
-              placeholder="Enter a CloudWatch Logs Insights query (run with Shift+Enter)"
-              portalOrigin="cloudwatch"
-            />
-          </div>
-          {ExtraFieldElement}
+  return (
+    <>
+      <QueryHeader
+        query={query}
+        onRunQuery={onRunQuery}
+        datasource={datasource}
+        onChange={onChange}
+        sqlCodeEditorIsDirty={false}
+      />
+      <LogGroupSelection datasource={datasource} query={query} onChange={onChange} onRunQuery={onRunQuery} />
+      <div className="gf-form-inline gf-form-inline--nowrap flex-grow-1">
+        <div className="gf-form gf-form--grow flex-shrink-1">
+          <QueryField
+            additionalPlugins={plugins}
+            query={query.expression ?? ''}
+            onChange={onChangeQuery}
+            onRunQuery={props.onRunQuery}
+            onTypeahead={onTypeahead}
+            cleanText={cleanText}
+            placeholder="Enter a CloudWatch Logs Insights query (run with Shift+Enter)"
+            portalOrigin="cloudwatch"
+          />
         </div>
-        {hint && (
```

Comment:
This was never used? :scream:

Reply:
Looks like it was way back in the beginning https://github.com/grafana/grafana/pull/23566
https://github.com/grafana/grafana/blob/58c4bdab14416488dd8d6c603d183a9896aecef5/public/app/plugins/datasource/cloudwatch/components/LogsQueryField.tsx#L229